meeting-minutes/2024-12-18.md

```diff
@@ -0,0 +1,134 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair)
+
+## 1.2 Introduction of participants/roll call (Co-Chair)
+
+Quorum requires participation of 9 or more of the 17 voting members (including the officers).
+
+| First Name | Last Name  | Company                                                     | Role(s)                   | Present |
+|:-----------|:-----------|:------------------------------------------------------------|:--------------------------|:--------|
+| Adrian     | Diglio     | Microsoft                                                   | Voting Member             | ??      |
+| David      | Kemp       | National Security Agency                                    | Member                    | ??      |
+| Denny      | Page       | Individual                                                  | Voting Member             | ???     |
+| Duncan     | Sparrell   | sFractal Consulting LLC                                     | Voting Member             | ??      |
+| Feng       | Cao        | Oracle                                                      | Member                    | ???     |
+| Harin      | Sarda      | Cisco Systems                                               | Voting Member             | ???     |
+| Jautau     | White      | Microsoft                                                   | Voting Member             | ???     |
+| Jeremy     | Rickard    | Microsoft                                                   | Member                    | ??      |
+| Justin     | Murphy     | DHS Cybersecurity and Infrastructure Security Agency (CISA) | Co-Chair                  | ???     |
+| Kris       | Vandecruys | Cisco Systems                                               | Voting Member             | ???     |
+| Kunal      | Modasiya   | Qualys, Inc.                                                | Member                    | ??      |
+| Langley    | Rock       | Dell                                                        | Voting Member             | ???     |
+| Martin     | Prpic      | Red Hat                                                     | Voting Member             | ??      |
+| Omar       | Santos     | Cisco Systems                                               | Co-Chair                  | ???     |
+| Pablo      | Quiroga    | Qualys, Inc.                                                | Voting Member             | ???     |
```

Comment:
Indeterminable whether Pablo still has voting rights.

Reply:
Need September minutes to determine Pablo's status.
If not present in September, would have lost voting rights after not being present for October meeting.